gov/governance/_GET_vote.gno

```diff
@@ -110,12 +110,9 @@ func GetVotedAtByVoteKey(voteKey string) uint64 {
 }
 
 func divideVoteKeyToProposalIdAndUser(voteKey string) (uint64, std.Address) {
-	parts := strings.Split(voteKey, ":")
-	if len(parts) != 2 {
-		panic(addDetailToError(
-			errInvalidInput,
-			ufmt.Sprintf("_GET_vote.gno__divideVoteKeyToProposalIdAndUser() || voteKey(%s) is invalid", voteKey),
```

Comment:
I really don't think removing detail error message is good idea.
It makes us hard to why it failed.
@onlyhyde Any thoughts?

Reply:
yes i agree with @r3v4s . error message is already confirmed our policy. so, we have to use previous error message

Reply:
reverted for panic messages

https://github.com/gnoswap-labs/gnoswap/pull/394/commits/1b17b6c4312417f01c59f084130cce786b482c45